roles/alertmanager_config/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Install Prometheus Alertmanager.
+  ansible.builtin.import_role:
+    name: cloudalchemy.alertmanager
+  vars:
+    alertmanager_config_file: "{{ alertmanager_config.alertmanager_config_file }}"
```

Comment:
This isn't currently set in `defaults/main.yml` ... I think if we set it to `/home/{{ _ce_provision.username }}/ce-provision/roles/alertmanager_config/templates/alertmanager.yml.j2` then we actually don't need *any* of the other stuff, because we can place our own template directly and pass in the vars in the `import_role` command.

Reply:
This is because their role, *unusually* but happily, allows for someone to change the path to the template file already.